modules/rosa-hcp-upgrading-cli-tutorial.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * upgrading/rosa-hcp-upgrading.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-upgrading-cli_{context}"]
+= Upgrading with the ROSA CLI
+
+You can manually upgrade a {hcp-title} cluster by using the ROSA CLI. This method schedules the cluster for an immediate upgrade if a more recent version is available.
+
+[NOTE]
+====
+Your control plane only supports machine pools within two minor Y-stream versions. For example, a {hcp-title} cluster with a control plane using version 4.15.z can support machine pools with version 4.13.z and 4.14.z, but the control plane will not support machine pools using version 4.12.z.
+====
+
+.Prerequisites
+
+* You have installed and configured the latest version of the ROSA CLI.
+
+.Procedure
+
+. Verify the current version of your cluster by running the following command:
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name|cluster_id> <1>
+----
+<1> Replace `<cluster_name|cluster_id>` with the cluster name or the cluster ID.
+
+. List the versions that you can use to upgrade your control plane and machine pools by running the following command:
++
+.. For the control plane versions, run the following command:
++
+[source,terminal]
+----
+$ rosa list upgrade --cluster=<cluster_name|cluster_id>
+----
++
+The command returns a list of upgradeable versions, including the recommended version.
++
+.Example output
++
+[source,terminal]
+----
+VERSION  NOTES
+4.14.8   recommended
+4.14.7
+4.14.6
+----
++
+.. For the machine pool versions, run the following command:
++
+[source,terminal]
+----
+$ rosa list upgrade --cluster <cluster-name> --machinepool <machinepool_name>
+----
++
+The command returns a list of upgradeable versions, including the recommended version.
++
+.Example output
++
+[source,terminal]
+----
+VERSION  NOTES
+4.14.5   recommended
+4.14.4
+4.14.3
+4.14.2
+4.14.1
+----
++
+[NOTE]
+====
+Your machine pools will not show upgrade versions beyond your cluster's current version. Ensure your cluster is up to date first.
```

Comment:
Is this meant to say that the latest available update for machine pools is only as high as the current version of the control plane? If so, I think a rewording like this will be clearer to users:
```suggestion
The latest available update for machine pools is limited to the current version of the control plane. Ensure your control plane is up to date first.
```
I'm not familiar enough with HCP to know if "control plane" or "cluster" are interchangeable in this case, so this is only a suggestion.